client/signup/config/step-components.js

```diff
@@ -17,7 +17,6 @@ const stepNameToModuleName = {
 	'domains-theme-preselected': 'domains',
 	'domains-launch': 'domains',
 	'from-url': 'import-url',
```

Comment:
Should we remove `from-url` and `import-url` here as well?

Reply:
@vishnugopal so we can remove also all: https://github.com/Automattic/wp-calypso/blob/54c36596cc7107b2c4e570505a128d80c2250947/client/signup/config/flows-pure.js#L334-L342

Reply:
Well I think `user` and `domains` are used elsewhere, I'm not entirely sure about `plans-import`. We should verify before we remove.

Reply:
(It's ok to leave this for a future PR if that makes more sense! I'll test this more thoroughly once you've marked this Ready for Review)

Reply:
@vishnugopal `from-url` is a blue page when you enter a Wix URL, it can be deleted. `domains` is also a blue page (as well as `plans-import`) but I think that it's not a good idea to remove it in order to avoid a potential domino effect. Maybe we just remove the parent `import` and leave the other child routes to the future PR which will delete all other remaining blue pages.

Those pages are outside the scope of this particular PR and AFAIK of the import section.

Reply:
Ack, makes sense @zaerl!